### DISCUSSION TITLE
How we can implement multiple virtual list which is using window scroll?.

### DISCUSSION THREAD

**Discussion by bipinrajbhar-rh** [TanStack/virtual#226] [Q&A]
How we can implement multiple virtual list which is using window scroll?.

https://github.com/bvaughn/react-virtualized/issues/1324#issuecomment-477683188

Hi @petersendidit , @piecyk , @mogelbrod , @oskarhane , @shastri9999.

**Comment by piecyk** [comment]
Yes it's possible, by writing a component using useVirtual with windowRef, then use it multiple times. https://codesandbox.io/s/multiple-window-scroll-j0sxc

There is a bug with getting correct value of scroll offset in context of dynamic size case, fixed by https://github.com/tannerlinsley/react-virtual/pull/227

**Reply by bipinrajbhar** [reply]
Thanks a lot.

**Reply by uffou** [reply]
@piecyk is it possible to use another element instead of the window as a scroll? And put multiple lists in it to share the scroll?

**Reply by ISilviu** [reply]
As far as I know, there are plans to add a prop where one can specify the scroll container: https://github.com/tannerlinsley/react-virtual/discussions/249.
